### ISSUE TITLE
Custom conversion

### ISSUE THREAD

**Issue by lbargaoanu** [agileobjects/ReadableExpressions#117]
```C#
var lambda = Expression.Lambda(Expression.MakeUnary(ExpressionType.Convert, 
    Expression.Default(typeof(object)), typeof(bool), typeof(Convert).GetMethod("IsDBNull")));
Console.WriteLine(lambda.ToReadableString());
```
I get `() => bool.IsDBNull(null)`, but a method call translation would work just fine.
I know it's weird, but I get a similar expression for a VB "cast" (`CInt`), which is, in fact, a method call to some VB-runtime function.

**Comment by SteveWilkes** [comment]
Thanks for the bug report - this will be fixed in the next release! :)

**Comment by SteveWilkes** [comment]
This is fixed as of v4.1 of the package, which is now [on NuGet](https://www.nuget.org/packages/AgileObjects.ReadableExpressions/4.1.0). Cheers!
